### PR TITLE
Remove shebangs from non-executable files

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

File: tools/brz_epydoc [hunk 1]
```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
```
Comment (jelmer) [inline tools/brz_epydoc:1]: This should be executable instead.

File: tools/time_graph.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```
Comment (jelmer) [inline tools/time_graph.py:1]: Same here.

**Review by jelmer — COMMENTED** [review]
LGTM, but there are two files that I think should be made executable rather than lose their shebang.

**Comment by hroncok** [comment]
> LGTM, but there are two files that I think should be made executable rather than lose their shebang.

Done.

**Comment by jelmer** [comment]
Forwarded to Launchpad. https://code.launchpad.net/~jelmer/brz/executability/+merge/394295

**Pull request closed without merging**